media-api/app/lib/elasticsearch/IsQueryFilter.scala

```diff
@@ -26,6 +27,8 @@ object IsQueryFilter {
       case s if s == s"$organisation-owned-illustration" => Some(IsOwnedIllustration(organisation))
       case s if s == s"$organisation-owned" => Some(IsOwnedImage(organisation))
       case "under-quota" => Some(IsUnderQuota(overQuotaAgencies()))
+      case "deleted" => Some(IsDeleted(true))
+      case "not-deleted" => Some(IsDeleted(false))
```

Comment:
can this be removed following https://github.com/guardian/grid/pull/3376/commits/cae0b648ef0804cbfc985cc52d9fd5d86d4adeeb

Reply:
@twrichards `not-deleted` removed and commits are been squashed